app/test.jsx

```diff
@@ -283,16 +296,24 @@ class App extends Component {
         this.setState({ hidePriceLines: evt.target.checked });
     };
 
-    onColorChange =(evt) => {
+    onShadeColorChange = (evt) => {
```

Comment:
also, I made some modification to `test.jsx` which binary2 needed and also make QA test easy